pages/dashboard.html

```diff
@@ -5,10 +5,267 @@
+            </div>
+
+            <div>
+                <img class="fundImg" src="../library/fund.png" alt="">
+                <label>Fund Transfer</label>
+            </div>
+
+            <div class="move">
+                <i class="bi bi-credit-card-2-back"></i>
+                <label>Cards</label>
+            </div>
+            <div>
+                <img class="fundImg" src="../library/Loan-Icon.png" alt="">
+                <label>Loans </label>
+            </div>
+            <div>
+                <img class="fundImg" src="../library/insurance-icons.png" alt="">
+                <label>Insurance</label>
+            </div>
+            <div>
+                <img class="fundImg" src="../library/bills.png.crdownload" alt="">
+                <label>Bills</label>
+            </div>
+
+        </menu>
+    </aside>
+
+    <section class="subtitle">
+        <label class="labelA">Account Summery</label>
+        <div class="container">
+
+            <div class="div1">
+                <img class="imgProfile" src="../library/profile.webp" alt="">
+            </div>
+
+            <div class="div2">
+                <label class="holderName">account Holder Name :</label>
+                <div class="moveName">
+                    <label>Rochi Jacline</label>
+                </div>
+
+            </div>
+
+
+            <div class="div3">
+                <label class="AccNum">Account Number:</label>
+                <div>
+                    <label>125315635</label>
+                </div>
+            </div>
+
+            <div class="div3">
+                <label class="AccNum">Branch:</label>
+                <div>
+                    <label> Haifa , Israel </label>
+                </div>
+            </div>
+
+            <div class="div3">
+                <label class="AccNum">Account Balance:</label>
+                <div>
+                    <label> 12,500 NIS </label>
+                </div>
+            </div>
+        </div> 
+
+        <div class="carde"> 
+
+            <div> 
+                <img class="charts" src="../library/image11.png" alt="">
+                
+            </div>
+
+
+
+        </div>
+
+    </section>  
+</div>
+
+    <main class="main">
+        <div class="tableT">
+            <label>Latest Transaction</label>
+        </div>
+
+    </main>
+
+    <section>
+
+        <form>
+            <table class="myTable">
+                <tr class="ttr">
+                    <th> <label>Narration</label></th>
+                    <th> <label>Date</label> </th>
+                    <th> <label>Type</label> </th>
+                    <th> <label> Closing Balance</label></th>
+                </tr> 
+                <tr></tr>
+                <tr></tr>
+                <tr></tr>
+                <tr></tr>
+                <tr></tr> 
+                <tr></tr>
+                <tr></tr>
+                <tr></tr>
+                <tr></tr> 
+                <tr></tr>
+                <tr></tr>
```

Comment:
unnecessary <tr> tags .. remove the tags and make a style in CSS for this table to get a space between them

Reply:
I have already fixed according to your requeste please check